msw/msw-app.sh

```diff
@@ -206,8 +206,16 @@ if [ "x$prototype_tk" = xfalse ] ; then
     cp -R $TK/bin $APP/
     cp -R $TK/lib $APP/
 
-    # remove some Tcl libraries Pd doesn't need
-    rm -rf $APP/lib/sqlite* $APP/lib/tdbc*
+    # remove bundled Tcl packges Pd doesn't need
+    rm -rf $APP/lib/itcl* $APP/lib/sqlite* $APP/lib/tdbc*
+
+    # install pthread from MinGW
+    if [ -v MINGW_PREFIX ] ; then
+        # MINGW_PREFIX should be either "/mingw32" or "/mingw64"
+        if [ -e $MINGW_PREFIX/bin/libwinpthread-1.dll ] ; then
+            cp -v $MINGW_PREFIX/bin/libwinpthread-1.dll $APP/bin
```

Comment:
this is not going to fly on cross-compilation builds. (e.g. on my Debian system, the file is `/usr/i686-w64-mingw32/lib/libwinpthread-1.dll` resp. `/usr/x86_64-w64-mingw32/lib/libwinpthread-1.dll`, so the `MINGW_PREFIX` on my system would arguably be something like `/usr/x86_64-w64-mingw32`

Reply:
i really don't know, but maybe we could use something calculated from the output of `$CC -print-libgcc-file-name`

Reply:
This is more of a convenience on Windows than a "works everywhere" solution. I figure advanced users doing a cross compile might be able to copy the dll themselves if they have to...

Reply:
Simple solution would be to try $MINGW_PREFIX/lib first, then $MINGW_PREFIX/bin second.

Reply:
Fixed.